pkgs/development/compilers/llvm/common/common-let.nix

```diff
@@ -12,14 +12,9 @@ rec {
   llvm_meta = {
     license =
       with lib.licenses;
-      [ ncsa ]
-      ++
-        # Contributions after June 1st, 2024 are only licensed under asl20 and
-        # llvm-exception: https://github.com/llvm/llvm-project/pull/92394
-        lib.optionals (lib.versionAtLeast release_version "19") [
-          asl20
-          llvm-exception
-        ];
+      # Contributions after June 1st, 2024 are only licensed under asl20 and
```

Comment:
Tangential but this comment was and is very misleading. The relevant info is that, since 19, there is asl20-only code, requiring acceptance of both licenses. Which one of those applies for new contributions is irrelevant.